chart/otomi/Chart.yaml

```diff
@@ -4,7 +4,7 @@ description: A Helm chart for installing otomi in Kubernetes
 home: https://otomi.io/
 icon: https://otomi.io/img/otomi-logo.svg
 type: application
-version: '0.1.6'
+version: '0.1.7-rc1'
```

Comment:
when the values change format this will become a breaking change since this value is mandatory...